Packs/VMRay/ReleaseNotes/1_1_6.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### VMRay
+- Added a bunch of unit tests.
```

Comment:
release notes should contain notes that impact our customer base - new features, bug fixes, etc. improved UT is important, but not of their concern.
```suggestion
```